app/models/recommendation/borough-board.js

```diff
@@ -0,0 +1,7 @@
+import CommunityBoardModel from './community-board';
+
+export default class RecommendationBoroughBoardModel extends CommunityBoardModel {
+// extension of CommunityBoardModel with no changes
+// CommunityBoardModel is an extension of RecommendationModel with added attributes
+
```

Comment:
This is missing the necessary vote attributes. It should look similar (if not the same as?) the Community Board one

Reply:
This is an extension of the Community Board model so it will already have all of the appropriate attributes included int he Community Board one